docs/tasks/includes/_rust-setup.md

```diff
@@ -0,0 +1,18 @@
+This is how to setup your code to use the Rust library.
+
+```rust
+use std::{
+    io::{Cursor, Write},
+    process::{Command, Stdio},
+};
+
+use anyhow::Result;
+use c2pa::{settings::load_settings_from_str, Builder, CallbackSigner, Reader};
+
+use c2pa_crypto::raw_signature::SigningAlg;
+use serde_json::json;
+
+const TEST_IMAGE: &[u8] = include_bytes!("../tests/fixtures/CA.jpg");
```

Comment:
It's not clear to me where to acquire these files. Should we provide a tarball for someone to download and extract?

Reply:
I added a "tip" about where these files are located in the Rust repo. Do you think we should change the path in the code to something else?  This code is copied from https://github.com/contentauth/c2pa-rs/blob/main/sdk/tests/v2_api_integration.rs#L33.